Makefile

```diff
@@ -86,6 +86,10 @@ mod-outdated: ## check outdated dependencies
 authors: ## generate AUTHORS file from git history
 	scripts/docs/generate-authors.sh
 
+.PHONY: completion
+completion: binary ## install the generated bash-completion script in the dev container
+	docker completion bash > /etc/bash_completion.d/docker
```

Comment:
does this need to be a separate make command?
Wondering if we could include the completion scripts (for all shells) inside the dev container by default.

Reply:
I was considering that, but we currently only have Bash installed. I think
we can consider adding more shells (but need to look at how much extra
time (and space) that would take by default to start the dev-container).

But yes, we can already split targets to prepare for that; I pushed an
update that splits the bash part to a separate make target :+1: PTAL